src/hooks/useAssetLogoSource.ts

```diff
@@ -1,9 +1,11 @@
-import TokenLogoLookupTable, { BAD_SRCS } from 'constants/TokenLogoLookupTable'
+import TokenLogoLookupTable from 'constants/TokenLogoLookupTable'
```

Comment:
File should be named `useTokenLogoSource` or default function should be renamed `useAssetLogoSource`.